listings_app/settings.py

```diff
@@ -11,6 +11,11 @@
 """
 
 import os
+import environ
+
+ROOT_DIR = environ.Path(__file__) - 2  # two folder back to root
```

Comment:
@chriswedgwood, another question please, why root is two folder back? the settings file is already at the root of the app, isn't it? or is root something else.

Reply:
I've added # (listings-app/listings_app/settings.py - 2 = listings-app/)
to the settings file
We need to go back to the root of the project as that's where the .env file will be

Reply:
oh god, I'm stupid, I had on my mind that settings file was on listing-app folder.
However still one more thing, sorry if it's stupid question again, settings is a file, not a folder, so shouldn't be listings-app/listings_app/ -1 = listing-app? or doesn't it work differently than other paths systems?

Reply:
No such thing as a stupid question in this project :)

The best way to check this in action is to create a python file in the same location as settings .py

Put this in it:

```
import environ

print(environ.Path(__file__)-2)
```
Then run it and fiddle with the 2 and see what comes back